src/Notification/AwardedBestAnswerBlueprint.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of fof/best-answer.
+ *
+ * Copyright (c) 2019 FriendsOfFlarum.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FoF\BestAnswer\Notification;
+
+use Flarum\Discussion\Discussion;
+use Flarum\Notification\Blueprint\BlueprintInterface;
+use Flarum\Notification\MailableInterface;
+use Flarum\User\User;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class AwardedBestAnswerBlueprint implements BlueprintInterface, MailableInterface
+{
+    /**
+     * @var Discussion
+     */
+    public $discussion;
+
+    /**
+     * @var TranslatorInterface
+     */
+    protected $translator;
+    protected $sender;
```

Comment:
Is this a leftover from some copy/paste? I don't see it used anywhere.

Reply:
Indeed it is, now removed